package.json

```diff
@@ -13,6 +13,8 @@
   "dependencies": {
     "react": "^16.8.0 || 17.x",
     "react-dom": "^16.8.0 || 17.x",
+    "react-jss": "^10.10.0",
```

Comment:
@gerardparareda I think that this might be an unused dependency, am I mistaken? 😕

Reply:
Nope, you are completely right. I missed this!